src/frontend/data/sideEffects/createLiveRegistration/index.spec.ts

```diff
@@ -0,0 +1,54 @@
+import fetchMock from 'fetch-mock';
+
+import { createLiveRegistration } from '.';
+
+jest.mock('../../appData', () => ({
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?